scala-package/mxnet-demo/java-demo/README.md

```diff
@@ -2,7 +2,13 @@
 This is an project created to use Maven-published Scala/Java package with two Java examples.
 ## Setup
 Please copy the downloaded MXNet Java package jar file under the `java-demo` folder.
+You can download the package from [here](https://repository.apache.org/#nexus-search;gav~org.apache.mxnet~~1.3.1-SNAPSHOT~~).
+Please download `1.3.1-SNAPSHOT` or [above version](https://repository.apache.org/#nexus-search;gav~org.apache.mxnet~~~~) to test to run this demo.
 
+Please make sure to rename the `jar` file you have download as the name that match the `MXNET_VERSION`, it looks similar as below:
```

Comment:
I think asking the user to manually download the JAR, and then rename the JAR is a hassle and we should avoid it - as notes above.
But putting that aside, note that the instruction to rename the JAR are missing from mxnet_java_on_intellij.md above